.github/workflows/docker.yaml

```diff
@@ -0,0 +1,58 @@
+name: Build & publish Docker image
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  publish_to_docker_hub:
+    runs-on: ubuntu-latest
+
+    env:
+      DOCKER_TAG: ${{ github.ref_name }}
+
+    steps:
+      - name: Dump GitHub context
+        id: github_context_step
+        run: echo '${{ toJSON(github) }}'
+
+      - name: Dump Input context
+        id: input_context_step
+        run: echo '${{ toJSON(inputs) }}'
+
+      - name: Checkout
+        uses: actions/checkout@v3
+      
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to DockerHub
```

Comment:
There’s a command that builds the image, this isn’t the way it’s supposed to be built